index.html

```diff
@@ -273,7 +285,45 @@
 Privacy therefore depends on having an equivalently strong countervailing force.
 
 
-## Exclusion
+### Age Verification {#age}
+
+A number of governments have chosen to mandate the use of age verification
+for access to certain online services.
+There are many issues that might be discussed on the topic,
+including whether age verification is even appropriate [[SUPPRESS]],
+all beyond the scope of this document.
+An upcoming workshop will address the issue
+of age-based restrictions on content and services
+in detail [[AGE-WS]].
```

Comment:
How about something like

```suggestion
The TAG is not confident that age verification is even appropriate ([[SUPPRESS]]),
but we are waiting for the outcome of the upcoming [[AGE-WS]] workshop
before taking a firm position.
```

I see that there's a risk in raising the question without spending the time to fully argue it, but I think that risk is worth it in order to warn about this risk to the web's architectural evolution.